### ISSUE TITLE
NTR: Eggshell calcification of lymph nodes

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#10620]
**Preferred term label:**
Eggshell calcification of lymph nodes

**Synonyms**
Peripheral calcification of lymph nodes

**Definition (free text, please give PubMed ID)**
PMID: 6779537; PMID: 29791561
Eggshell calcification refers to fine calcification seen at the periphery of a mass and usually relates to lamellar lymph node calcification. eggshell calcification, commonly occurs in patients with silicosis and coal-worker's pneumoconiosis. Sarcoidosis, postirradiation Hodgkin disease, blastomycosis, and scleroderma are other reported causes. Two not previously described causes, amyloidosis and histoplasmosis, are presented, and disorders that simulate eggshell calcification are listed. The explanation for the eggshell pattern of calcification is not known. 

**Parent term (use hpo.jax.org/app)**
Abnormal lymph node morphology HP:0002733

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0003-0113-912X, 0000-0002-7371-8158

**Comment by pnrobinson** [comment]
@GrieseAG 
Adding new term: 
- ID: HP:6001019
- label: Eggshell calcification of lymph nodes
- parents: [Abnormal lymph node morphology (HP:0002733)](https://hpo.jax.org/app/browse/term/HP:0002733)
- pmid: PMID:6779537|PMID:29791561
- def: Eggshell calcification refers to fine calcification seen at the periphery of a mass and usually relates to lamellar lymph node calcification.
- comment: Eggshell calcification, commonly occurs in patients with silicosis and coal-worker's pneumoconiosis. Sarcoidosis, postirradiation Hodgkin disease, blastomycosis, and scleroderma are other reported causes. Eggshell calcification has more rarely been observed in amyloidosis and histoplasmosis.
- orcid: orcid:0000-0002-7371-8158
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10620
- synonyms: Peripheral calcification of lymph nodes(NONE)
